keras_retinanet/bin/train.py

```diff
@@ -467,16 +498,34 @@ def main(args=None):
         prediction_model,
         validation_generator,
         args,
+        tensorboard_image=writer,
     )
 
     # start training
-    training_model.fit_generator(
+    # start training
+    if args.val_loss:
+        
+        training_model.fit_generator(
+            generator=train_generator,
+            steps_per_epoch=args.steps,
+            validation_data=validation_generator,
```

Comment:
I think this can be shortened by doing:

`validation_steps=validation_generator if args.val_loss else None`

Then the `if` is not required.